test/battle/move_effect_secondary/order_up.c

```diff
@@ -0,0 +1,116 @@
+#include "global.h"
+#include "test/battle.h"
+
```

Comment:
```suggestion

ASSUMPTIONS
{
    ASSUME(gMovesInfo[MOVE_ORDER_UP].additionalEffects[0].moveEffect == MOVE_EFFECT_ORDER_UP);
}

```
For safety I guess.

Reply:
Assumption was put in Commander file instead of Order Up file btw